src/transformers/trainer.py

```diff
@@ -2455,7 +2465,7 @@ def _inner_training_loop(
                         self.control = self.callback_handler.on_step_begin(args, self.state, self.control)
 
                     with self.accelerator.accumulate(model):
-                        tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
+                        tr_loss_step = self.training_step(model, inputs)
```

Comment:
Why `num_items_in_batch` is removed here?

Reply:
Leftover artifact while I was experimenting